layouts/main.tsx

```diff
@@ -112,6 +115,14 @@ const Layout = ({ children, title = "Livepeer Explorer" }) => {
     [currentRound?.currentL1Block, pollData]
   );
 
+  const totalActiveProposals = useMemo(() => {
+    const currentBlock = currentRound?.id ? parseInt(String(currentRound.id)) : 0;
+    return proposalsData?.treasuryProposals.reduce((count, p) => {
+      const voteEndBlock = Number(p.voteEnd);
+      return voteEndBlock > currentBlock ? count + 1 : count;
```

Comment:
Oh btw, one cosmetic change: These are "rounds", not "blocks", so I'd rename these variables to avoid confusion.